src/components/Forms/CommentBox/CommentBox.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { CommentBox } from './CommentBox';
+
+storiesOf('Form', module)
+  .add('CommentBox',
+    withInfo({
+      text: 'description: Component for commenting posts',
+      inline: false,
+      source: false,
+    })(() => (
+      <>
+        <h2>Sample post</h2>
+        <CommentBox />
+      </>
+    )),
+    {
+      notes: 'note',
```

Comment:
professor complaint that we don't have enough explanation for each component in front end document

Reply:
I added more info